Assets/Scripts/Game/UserInterfaceWindows/DaggerfallInventoryWindow.cs

```diff
@@ -957,6 +1053,43 @@ protected void UpdateAccessoryItemsDisplay()
         #endregion
 
         #region Private Methods
+        protected void PopulateItemGroupNames()
+        {
+            itemGroupNames[0] = "drugs";
+            itemGroupNames[1] = "uselessitems1";
+            itemGroupNames[2] = "armor";
+            itemGroupNames[3] = "weapons";
+            itemGroupNames[4] = "magicitems";
```

Comment:
These values are stored in a static field

```cs
protected static string[] itemGroupNames = new string[30];
```

so you don't need to initialize them every time the window is opened. You can also use in-line initialization.

```cs
protected static string[] itemGroupNames = new string[]
{
    "drugs",
    "uselessitems1",
    // ---
}
```